src/libs/actions/Session/index.ts

```diff
@@ -69,6 +69,9 @@ const INVALID_TOKEN = 'pizza';
 
 let session: Session = {};
 let authPromiseResolver: ((value: boolean) => void) | null = null;
+
+let isSwitchedAccount = false;
```

Comment:
```suggestion
let hasSwitchedAccountInHybridMode = false;
```
I think this is just a touch clearer, since it's always switching an account